src/js/control-bar/text-track-controls/captions-button.js

```diff
@@ -26,7 +26,7 @@ class CaptionsButton extends TextTrackButton {
    */
   constructor(player, options, ready) {
     super(player, options, ready);
-    this.el_.setAttribute('aria-label', 'Captions Menu');
+    this.el_.setAttribute('aria-label', this.localize('Captions Menu'));
```

Comment:
@OwenEdwards thoughts on whether we need to create a new item for this or if we can just re-use `Captions` and `Chapters` etc for these button labels?